processor/tailsamplingprocessor/config.go

```diff
@@ -62,6 +64,33 @@ type SubPolicyCfg struct {
 	StringAttributeCfg StringAttributeCfg `mapstructure:"string_attribute"`
 	// Configs for rate limiting filter sampling policy evaluator.
 	RateLimitingCfg RateLimitingCfg `mapstructure:"rate_limiting"`
+	// Configs for latency filter sampling policy evaluator.
+	LatencyCfg LatencyCfg `mapstructure:"latency"`
+	// Configs for status code filter sampling policy evaluator.
+	StatusCodeCfg StatusCodeCfg `mapstructure:"status_code"`
+	// Configs for and policy evaluator.
+	AndCfg AndCfg `mapstructure:"and"`
+}
+
+type AndSubPolicyCfg struct {
```

Comment:
This seems to be missing the probabilistic config. Is that on purpose? Also, does it make sense to have nested AndPolicyCfgs ?

Reply:
I can't think of a good use case for nested `and` policies but I do think it makes sense to have a `composite` policy with an `and` sub policy